src/styles/Widget.scss

```diff
@@ -0,0 +1,26 @@
+.onsite-carousel {
+  display: relative;
+  // Sum of Product.scss ems
+  height: 23.5em;
+
+  @mixin onsite-arrow($size, $left) {
+    position: absolute;
+    z-index: 1;
+    height: $size;
+    margin-top: -($size)/2;
+    top: 50%;
+    @if $left == true { 
+      left: 0px;
+    } @else {
+      right: 0px;
+    }
+  }
+
+  .slick-arrow.slick-prev {
```

Comment:
vai ser complicado deixar isso aqui, isso não é estilo do teu componente e sim usando pelo slick.
e se o cliente usar um slick no site dele e fizer a mesma coisa?
como tu vai garantir que teu componente não vai quebrar por causo do cliente?
quando tu usar o CssModule isso não vai funcionar!

Reply:
Lido com isso fazendo aninhamento, apenas sera aplicado no q for onsite-carousel.